### PR TITLE
428 cda fhir engine first cut

### PR DESCRIPTION
**Pull request by Goncharo**
Ref: #_[ticket-number]_

### Dependencies

N/A

### Description

CDA to FHIR engine first cut. CDK/infra updates are TODO

### Release Plan

asap

### CODE REVIEW

File: api/app/src/sequelize/migrations/2023050403101805-update-patient-data.ts [hunk 1]
```diff
@@ -22,8 +22,10 @@ export const up: Migration = async ({ context: queryInterface }) => {
       }
       patient.data = JSON.stringify(patient.data);
     }
-    await queryInterface.bulkDelete(patientTableName, {}, { transaction });
-    await queryInterface.bulkInsert(patientTableName, patients, { transaction });
+    if (patients.length > 0) {
+      await queryInterface.bulkDelete(patientTableName, {}, { transaction });
+      await queryInterface.bulkInsert(patientTableName, patients, { transaction });
+    }
```
Comment (leite08) [inline api/app/src/sequelize/migrations/2023050403101805-update-patient-data.ts:28]: Why are we changing past migrations?

**Review by leite08 — COMMENTED** [review]
Left comments/questions about our API.

Fhir engine was too large and I didn't review.

Question/suggestion: we're calling it FHIR Engine. I think this can be confusing... "Is this the FHIR server?" What if we call it what it is, `CdaToFhir`, or the original project name?

File: api/app/src/routes/medical/tmp-data [hunk 2]
```diff
@@ -0,0 +1,3102 @@
+<?xml version="1.0"?>
```
Comment (Orta21) [inline api/app/src/routes/medical/tmp-data:1]: Is this leftover?

**Review by Orta21 — APPROVED** [review]
Besides Rafs suggestions i think it looks good. Also assume we dont need to check out fhir-engine code

**Pull request closed without merging**